integration-tests/config-key/src/test/java/io/quarkiverse/openapi/generator/configkey/QuarkusConfigKeyOpenApiTest.java

```diff
@@ -0,0 +1,50 @@
+package io.quarkiverse.openapi.generator.configkey;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import jakarta.inject.Inject;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.openapi.quarkus.config_key_openapi_yaml.api.ReactiveGreetingResourceApi;
+import org.openapi.quarkus.empty_config_key_yaml.api.DefaultApi;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.smallrye.mutiny.Uni;
+
+@QuarkusTest
+@Tag("resteasy-reactive")
+class QuarkusConfigKeyOpenApiTest {
+
+    @RestClient
+    @Inject
+    ReactiveGreetingResourceApi api;
+
+    @RestClient
+    @Inject
+    DefaultApi defaultApi;
```

Comment:
This is not used.

Reply:
Hi @hbelmiro, I am using it to test the config-key empty behavior.